erpnext/regional/india/e_invoice/utils.py

```diff
@@ -86,10 +86,10 @@ def get_doc_details(invoice):
 		invoice_date=invoice_date
 	))
 
-def get_party_details(address_name):
+def get_party_details(address_name, company_address=None, billing_address=None, shipping_address=None):
```

Comment:
- Let's remove unused parameters here. Only `shipping_address` should suffice.
- Will need to fix the call to this function as well.
- Might need changes in other branches as well.

Reply:
I am working on re-writing the validations with bulk e-invoice generation PR. So, we could keep this as it is? This will get overwritten in a few weeks. 

Or should I just remove these for now?

Reply:
Your choice. As long as you'll fix it in a few weeks.